tests/test_dbt.py

```diff
@@ -54,7 +54,7 @@ def test_get_datadiff_variables_empty(self):
     def test_get_models(self, mock_manifest_parser, mock_run_parser, mock_open):
         expected_value = "expected_value"
         mock_self = Mock()
-        mock_self.project_dir = ""
+        mock_self.project_dir = Path()
```

Comment:
I think we're missing an import

```
NameError: name 'Path' is not defined
```

Reply:
Thanks @dlawin -- just pushed a commit that adds the import.